corehq/apps/userreports/filters/factory.py

```diff
@@ -71,7 +71,7 @@ def _build_boolean_expression_filter(spec, factory_context):
 def _build_named_filter(spec, factory_context):
     wrapped = NamedFilterSpec.wrap(spec)
     try:
-        filter = factory_context.named_filters[wrapped.name]
+        filter = factory_context.get_named_filter(wrapped.name)
     except KeyError as e:
```

Comment:
Does `get_named_filter` raise `KeyError` (as the dict lookup did) when the lookup fails? I think it does not.

Reply:
Nope: 51c8572a3d643a292262197a05716c17476d8726